pulp-glue/pulp_glue/rpm/context.py

```diff
@@ -197,6 +203,16 @@ def preprocess_entity(self, body: EntityDefinition, partial: bool = False) -> En
         if version is not None:
             repository_href = body.pop("repository")
             body["repository_version"] = f"{repository_href}versions/{version}/"
+        if "repo_config" in body:
+            self.pulp_ctx.needs_plugin(
+                PluginRequirement(
+                    "rpm",
+                    specifier=">=3.24.0",
+                    feature=_("A JSON document describing config.repo file."),
+                )
+            )
+        elif self.pulp_ctx.has_plugin(PluginRequirement("rpm", specifier="<3.24.0")):
+            body.pop("repo_config", None)
```

Comment:
This part is dead code. It's only reached, when repo_config wasn't in body in the first place.

Reply:
you're right!